vms/platformvm/state/camino_stakers.go

```diff
@@ -105,3 +105,23 @@ func writeDeferredDiff(
 	}
 	return nil
 }
+
+func (cs *caminoState) SetValidatorUptime(subnetID ids.ID, nodeID ids.NodeID, uptime time.Duration) {
+	if cs.modifiedUptimes[nodeID] == nil {
+		cs.modifiedUptimes[nodeID] = make(map[ids.ID]time.Duration)
+	}
+	cs.modifiedUptimes[nodeID][subnetID] = uptime
+}
+
+func (cs *caminoState) writeValidatorUptimes() error {
```

Comment:
all "write" funcs are writing state to mem
but this one seems to only modify state uptimes in-mem struct - it does not guarantee that this will be written into db
imo, thats wrong